libraries/lib-string-utils/3party/fast_float.h

```diff
@@ -0,0 +1,2947 @@
+// fast_float by Daniel Lemire
```

Comment:
I'm taking this whole file on trust.  Are there any differences from the original I should know?  As there were with ToChars?

Reply:
This time it is just the latest release of the library!